### PR TITLE
feat: support of reakit orientation option on tabs

### PR DESCRIPTION
**Pull request by guillaumewttj**
Capture d’écran 2020-06-30 à 17 40 07

### CODE REVIEW

File: packages/Tabs/index.js [hunk 1]
```diff
@@ -70,7 +85,8 @@ export const TabList = React.forwardRef(({ as, children, ...props }, ref) => {
 TabList.displayName = 'TabList'
 TabList.propTypes /* remove-proptypes */ = {
   as: oneOfType(COMPONENT_TYPE),
-  children: node
+  children: node,
+  orientation: oneOf(['veritcal', 'horizontal'])
```
Comment (theo-mesnil) [inline packages/Tabs/index.js:89]:
```suggestion
  orientation: oneOf(['vertical', 'horizontal'])
```

File: packages/Tabs/index.js [hunk 2]
```diff
@@ -20,48 +21,62 @@ function useTrackActiveTabs(state, children) {
...
 ActiveBar.displayName = 'ActiveBar'
 ActiveBar.propTypes /* remove-proptypes */ = {
   activeTab: oneOfType(COMPONENT_TYPE),
-  listRef: shape({ current: any })
+  listRef: shape({ current: any }),
+  orientation: oneOf(['veritcal', 'horizontal'])
 }
 
-export const TabList = React.forwardRef(({ as, children, ...props }, ref) => {
+export const TabList = React.forwardRef(({ as, children, orientation, ...props }, ref) => {
```
Comment (theo-mesnil) [inline packages/Tabs/index.js:67]: horizontal by default ?

File: packages/Tabs/index.js (hunk 1 above)
Comment (Calvein) [reply packages/Tabs/index.js:89]: Name: Code Spell Checker
Id: streetsidesoftware.code-spell-checker
Description: Spelling checker for source code
Version: 1.9.0
Publisher: Street Side Software
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

😛

File: packages/Tabs/index.js [hunk 3]
```diff
@@ -20,48 +21,62 @@ function useTrackActiveTabs(state, children) {
...
+const ActiveBar = ({ activeTab, listRef, orientation }) => {
+  const activeBar = useActiveBarState(listRef, activeTab, orientation)
   return <S.ActiveBar {...activeBar} />
 }
 ActiveBar.displayName = 'ActiveBar'
 ActiveBar.propTypes /* remove-proptypes */ = {
   activeTab: oneOfType(COMPONENT_TYPE),
-  listRef: shape({ current: any })
+  listRef: shape({ current: any }),
+  orientation: oneOf(['veritcal', 'horizontal'])
```
Comment (Calvein) [inline packages/Tabs/index.js:64]:
```suggestion
  orientation: oneOf(['vertical', 'horizontal'])
```

File: packages/Tabs/index.js (hunk 2 above)
Comment (guillaumewttj) [reply packages/Tabs/index.js:67]: We don't need to have a default value here since it's handled by the `useTabState` util from reakit

Comment (theo-mesnil) [reply packages/Tabs/index.js:67]: Ok, it's was for the documentation of properties :)

**Pull request merged**